README.md

```diff
@@ -37,6 +37,18 @@ By default, this will run tests. If you want to skip running tests use
 
     $ mvn install -DskipTests
 
+The build is stable starting with Maven version `3.6.x` or higher. 
+With a lower version you will get errors when trying to resolve dependencies created in the Antenna build.
+A possible error message would look like this:
+
+```  
+Failed to process POM for org.eclipse.sw360.antenna:model:jar:1.0.0-SNAPSHOT: 
+Non-resolvable parent POM for org.eclipse.sw360.antenna:model:1.0.0-SNAPSHOT: 
+Could not find artifact org.eclipse.sw360.antenna:antenna-management:pom:${revision} 
+in central (https://repo.maven.apache.org/maven2/) and 'parent.relativePath' points at wrong local POM
+[ERROR] org.eclipse.sw360.antenna:model:jar:1.0.0-SNAPSHOT
+```
+
 #### Optional Profiles
```

Comment:
Looks good. I found this link: https://maven.apache.org/maven-ci-friendly.html which describes the usage of the ${revision} property. It says that the property is available "Starting with Maven 3.5.0-beta-1". So probably Maven 3.5.x will be sufficient.